code/datums/wounds/_wounds.dm

```diff
@@ -89,7 +85,7 @@
 	var/cryo_progress
 
 	/// What kind of scars this wound will create description wise once healed
-	var/list/scarring_descriptions = list("general disfigurement")
+	var/scar_keyword = "generic"
```

Comment:
Could you turn this into defines?

Reply:
And this one

Reply:
Wouldn't it be less helpful since that keyword is only used in the .json file containing the descriptions, where the define wouldn't reach?